build.gradle

```diff
@@ -5,7 +5,7 @@ group 'org.zowe.explorer.files'
 
 buildscript {
     ext {
-        licenseGradlePluginVerion = '0.14.0'
+        licenseGradlePluginVersion = '0.13.1'
```

Comment:
Why the downgrade?

Reply:
Integration tests (LPTA) fail when you upgrade springBootVersion to 2.7.12. You need to have `licenseGradlePluginVersion` set to 0.13.1 for all tests to pass.